### PR TITLE
Add initial Portland 2024 site

### PR DESCRIPTION
**Pull request by ericholscher**
* Blog post: https://writethedocs-www--2041.org.readthedocs.build/conf/portland/2024/news/welcome/
* Website: https://writethedocs-www--2041.org.readthedocs.build/conf/portland/2024/

----
:books: Documentation preview :books:: https://writethedocs-www--2041.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/portland/2024/news/welcome.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+We are excited to welcome everyone back to Revolution Hall in Portland for another year of great talks and community building.
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+**We plan to launch our CFP in November** for those who are excited to speak.
+We're still working out the exact details for the CFP,
```
Comment (thatdocslady) [inline docs/conf/portland/2024/news/welcome.rst]: Does this mean pre-record all the talks and broadcast them on the stage? I thought that we decided not to do that, but instead go back to recording the live talks and publishing them. Not sure if I missed this change or if this paragraph is leftover from a previous iteration...

File: docs/conf/portland/2024/news/welcome.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+and invite speakers who cannot or do not feel comfortable attending in person.
+
+Start thinking about your talk proposal now,
+and we'll let you know as soon as the CFP opens.
+
+
+Tickets
+-------
+
+**Tickets will go on sale in {{ date.tickets_live }}**.
```
Comment (thatdocslady) [inline docs/conf/portland/2024/news/welcome.rst:29]: I remember every year there are always some people who need to use their travel budget before the year is over, so maybe if we open CFP in November we could open ticket sales in December?

**Review by thatdocslady — COMMENTED** [review]
Looks great, a few nit-picky questions :)

File: docs/conf/portland/2024/news/welcome.rst (hunk 1 above)
Comment (ericholscher) [reply docs/conf/portland/2024/news/welcome.rst]: Probably leftover, but I think details here just mean the dates, etc.? 

cc @plaindocs if you have a strong preference on timing. It's earlier this year so we're gonna run into the holidays in some fashion :/

File: docs/conf/portland/2024/news/welcome.rst (hunk 2 above)
Comment (ericholscher) [reply docs/conf/portland/2024/news/welcome.rst:29]: Yea, seems reasonable? Overall it's a bit annoying to open tickets in the prior year, but not a huge issue.

File: docs/conf/portland/2024/news/welcome.rst (hunk 1 above)
Comment (plaindocs) [reply docs/conf/portland/2024/news/welcome.rst]: Yeah, that works. The closing is more important than the opening for me. @thatdocslady live talks only.

File: docs/_data/portland-2024-config.yaml [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+
+  social_venue: TBD
+  venue: Revolution Hall
+  photos: TBD
+  mainroom: Main stage
+  unconfroom: Library & Astoria room (Bottom floor)
+  projector_ratio: "16:9"
+  job_fair_room: Sponsor area
+
+cfp:
```
Comment (plaindocs) [inline docs/_data/portland-2024-config.yaml:165]: Looks workable.

**Review by plaindocs — APPROVED** [review]
@ericholscher looks good to me.

What was your objection to the slack check? That it wasn't working as planned, or that you disagreed with the rule?

**Review by thatdocslady — APPROVED** [review]

**Comment by ericholscher** [comment]
@plaindocs It was failing on what looked like the actual links to the pages (eg. `:doc:/slack`)

**Comment by plaindocs** [comment]
> @plaindocs It was failing on what looked like the actual links to the pages `(eg. `:doc:/slack`)

bah. It's not super important anyway, but it is mildly odd.

**Pull request merged**